doc/browsing_cubes/intro.rst

```diff
@@ -1,9 +1,54 @@
 Browsing Cubes
 ==============
 
-This is a description of how you can use the cube_browser package to look at cubes and make plots and sliders without having to make any widgets.
+The Cube Browser package offers an API which you can use to visualize your cubes in a Jupyter notebook.
+It utilizes the Python, Matplotlib, Iris and Cartopy libraries while offering the additional functionality of plotting an entire cube without having to slice it.
 
-Links to Notebooks
-------------------
+You can make use of the Cube Browser library to write a short notebook to display your Iris cube data.
+There are several advantages of using this approach, but the main advantage is flexibility in the options you can apply to each plot.
```

Comment:
I think simplicity is another key thing of this - you can with (relative - one needs to be careful about using superlatives in documentation in case a user of the documentation finds a bit that you've described as easy to be really hard and gets put off) ease create a template that is easily reusable. I'm going to pick up on this thought just further down too.